contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/BrowsingCard.vue

```diff
@@ -246,4 +260,19 @@
     margin-bottom: -3px;
   }
 
+  .metadata {
+    color: var(--v-grey-darken2);
```

Comment:
Just to be sure - CSS custom properties [are not supported in IE11](https://caniuse.com/#feat=css-variables) (Vuetify is supported in IE11 with some polyfills). Are we okay with that?

Reply:
We currently block IE11, so this should be okay